config/versions.json

```diff
@@ -1,5 +1,5 @@
 {
     "$schema": "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/deployment/config/versions/3-0-0.json",
     "spack": "0.22",
-    "spack-packages": "2025.07.005"
+    "spack-packages": "327-babeltrace2"
```

Comment:
This change can be removed once @CodeGat updates `build-cd` to use the latest Spack v0.22 commit.